### PR TITLE
add instructions to log into the registry

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@adellape @ncdc ptal.

**Comment by ncdc** [comment]
:+1:

File: admin_guide/docker_registry.adoc [hunk 1]
```diff
@@ -230,7 +248,7 @@ $ sudo systemctl restart docker
 ----
 ====
 +
-. Validate the docker client connection. Running `docker push` to the registry or `docker pull` from the registry should succeed:
+. Validate the docker client connection. Running `docker push` to the registry or `docker pull` from the registry should succeed.  Make sure you have link:#access]logged into the registry].
```
Comment (adellape) [inline admin_guide/docker_registry.adoc]: s/]logged/[logged/

**Comment by bparees** [comment]
bracket reticulated.

**Comment by adellape** [comment]
Thanks, LGTM

**Pull request merged**